src/Main/Main.js

```diff
@@ -29,12 +31,16 @@ class Test extends Component {
   }
 
   render() {
-    let RightSide = <ViewRoom />
-    let StickyTitle = "View Room"
+    let RightSide = <HomePage />
```

Comment:
Noticed a few places where semi-colons are missing, most have been previously missed and are just lines you have edited but do you mind adding them to this PR?

Lines 34, 35, 37, 38, 41, 42 are the ones I've noticed but there may be others.